frontend/src/components/modal.tsx

```diff
@@ -97,12 +101,22 @@ export function Modal({
             transitionClassName,
           )}
         >
-          <Dialog ref={dialogRef}>
+          <DialogComponent ref={dialogRef}>
             <ModalTitle id="modal-title">{label}</ModalTitle>
             {children}
-          </Dialog>
+          </DialogComponent>
         </div>
       </div>
     </div>
   );
 }
+
+export function DoubleWidthModal(
```

Comment:
Could I have something better here?
(variable width dialogs would be very nice)